cmd/terrad/testnet.go

```diff
@@ -118,8 +118,8 @@ func InitTestnet(
 		chainID = "chain-" + tmrand.NewRand().Str(6)
 	}
 
-	nodeIDs := make([]string, numValidators)
-	valPubKeys := make([]cryptotypes.PubKey, numValidators)
+	nodeIDs := make([]string, 6)
```

Comment:
you shouldn't hardcore number of node like this. It makes future change hard.

Reply:
The idea is to make 6 testnet nodes of which a variable number are validators. I can put the 6 into a constant and use that instead. Would that resolve the Convo?

Reply:
Or we could pass another parameter `numNodes`